egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/report/BankAdviceReportAction.java

```diff
@@ -467,11 +473,13 @@ public String exportPDF() {
         final Map<String, Object> reportParams = new HashMap<String, Object>();
         final StringBuffer letterContext = new StringBuffer();
         letterContext
-                .append("             I request you to transfer the amount indicated below through RTGS duly debiting from the")
+                .append("             I request you to transfer the amount indicated below through RTGS/NEFT duly debiting from the")
                 .append("  Current Account No: ")
                 .append(getBankAccountNumber(bankaccount.getId()) != null ? getBankAccountNumber(bankaccount.getId()) : " ")
                 .append("  under your bank to the following bank accounts:");
         reportParams.put("bankName", getBankName(bank.getId()));
+        final City city = cityService.getCityByCode(ApplicationThreadLocals.getCityCode());
```

Comment:
City Name also present in ApplicationThreadLocals. Read from Thread locals if it is present.

Reply:
i am reading from ApplicationThreadLocals.getCityCode() only